src/components/MonacoEditor/MonacoEditor.jsx

```diff
@@ -66,7 +66,7 @@ export function MonacoEditor({ value, onChange, onSave }) {
 
     editor.addAction(createFormatAction(monaco));
     editor.addAction(createSaveAction(monaco, onSave));
-  }, []);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Ser ut som du disabler denne sjekken mange steder. Hvorfor trenger vi det?
Virker litt som vi bruker useeffect feil om den må disables så mange steder?

Reply:
En del av denne oppgraderinga er eslint fra 5 til 7. Og en ny standardregel er dette. 
Om eg fjerner kommentaren klager eslint slik:
`React Hook useEffect has missing dependencies: 'onChange', 'onSave', and 'value'. Either include them or remove the dependency array.`. Anbefalinga no er å lage funksjoner med useCallback hook og så sende denne til useEffect. Dok https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies

Reply:
Å berre legge til onChange, onSave og value i lista funker ikkje, for då kjøres hooken for kvar endring i value og denne skal kun kjøres en gong. Så då var det kjappere å legge på kommentar enn å skrive om alle hooks.